### PR TITLE
Added sentry instructions

### PR DESCRIPTION
**Pull request by Parth4337**
Added instructions for setting up sentry structure and fixed some typos

### CODE REVIEW

File: docs/testnet/onomy-testnet-docs/setting-up-a-validator.md [hunk 1]
```diff
@@ -153,7 +158,7 @@ onomyd --home $HOME/.onomy tx bank send <your validator address> <your delegate
 ```
 2. Using faucet command, from the Onomy-side faucet:
 ```
-curl -X POST http://testnet1.onomy.io:8000/ -H  "accept: application/json" -H  "Content-Type: application/json" -d "{  \"address\": \"<your delegate cosmos address>\",  \"coins\": [    \"10000000nom\"  ]}"
+curl -X POST http://testnet1.onomy.io:8000/ -H  "accept: application/json" -H  "Content-Type: application/json" -d "{  \"address\": \"<your delegate cosmos address>\",  \"coins\": [    \"100000000nom\"  ]}"
```
Comment (dzmitryhil) [inline docs/testnet/onomy-testnet-docs/setting-up-a-validator.md:161]: For the next testnet the main currency will be "anom", so we need to change it for all scripts and docs

File: docs/testnet/onomy-testnet-docs/pre-installation.md [hunk 2]
```diff
@@ -26,24 +25,26 @@ export PATH=$PATH:$HOME/.onomy/bin
 `
```
Comment (dzmitryhil) [inline docs/testnet/onomy-testnet-docs/pre-installation.md:25]: Check markdown link check it is red
https://github.com/onomyprotocol/onomy/runs/4279388673?check_suite_focus=true

**Review by dzmitryhil — CHANGES_REQUESTED** [review]

File: docs/testnet/onomy-testnet-docs/pre-installation.md (hunk 2 above)
Comment (Parth4337) [reply docs/testnet/onomy-testnet-docs/pre-installation.md:25]: Done

File: docs/testnet/onomy-testnet-docs/setting-up-a-validator.md (hunk 1 above)
Comment (Parth4337) [reply docs/testnet/onomy-testnet-docs/setting-up-a-validator.md:161]: Changed, Check once

**Comment by cgdusek** [comment]
Why are these test indicator boxes red and I see red X ever commit?

**Pull request merged**